backend-rust/src/graphql_api.rs

```diff
@@ -115,7 +116,12 @@ pub struct ApiServiceConfig {
 }
 
 #[derive(MergedObject, Default)]
-pub struct Query(BaseQuery, AccountMetricsQuery, TransactionMetricsQuery);
+pub struct Query(
+    BaseQuery,
+    AccountMetricsQuery,
+    TransactionMetricsQuery,
+    block_metrics::BlockMetricsQuery,
```

Comment:
nitpicking to be consistent.

```
    BaseQuery,
    AccountMetricsQuery,
    TransactionMetricsQuery,
    BlockMetricsQuery,
```

Reply:
Right, I'll make these consistent in the next PR